specification.md

```diff
@@ -560,6 +560,14 @@ in the future such that old Agents automatically report that they don't
 support the new capability.
 This field MUST be always set.
 
+This field MUST be set in the first AgentToServer sent by the Agent and MAY be
+omitted in subsequent AgentToServer messages by setting it to
+UnspecifiedAgentCapability value. An Agent MAY update its'
+AgentCapabilities.AcceptsRemoteConfig capability at ANY time after the first
```

Comment:
Can we make it even more explicit that no other capabilities may change in subsequent messages?